lib/puma/binder.rb

```diff
@@ -50,7 +50,8 @@ def env(sock)
 
     def close
       @ios.each { |i| i.close }
-      @unix_paths.each { |i| File.unlink i }
+      # Errno::ENOENT is intermittently raised
+      @unix_paths.each { |i| File.unlink(i) rescue Errno::ENOENT }
```

Comment:
please turn this into a multi-liner. doing this one line is equivalent to `rescue nil`.

Reply:
That was stupid of me.  Apologies.
Thanks for pointing it out...